ext/templates/puppetdb-ssl-setup.erb

```diff
@@ -226,7 +226,7 @@ else
   agent_confdir=`puppet agent --configprint confdir`
   agent_vardir=`puppet agent --configprint vardir`
 
-  if [ -d "/etc/puppetlabs/puppetdb" ] ; then
+  if id -u pe-puppetdb > /dev/null 2>&1; then
```

Comment:
@wkalt we probably want to check it works on all the distros we care about, we have no tests that will shake this path up, pe doesn't use ssl-setup during installation although some users use it standalone for support reasons I believe.